tests/baselines/reference/functionsMissingReturnStatementsAndExpressions.errors.txt

```diff
@@ -40,12 +40,12 @@ tests/cases/compiler/functionsMissingReturnStatementsAndExpressions.ts(127,5): e
         return null;
     }
     
-    function f8(): void {
+    function f8(): any {
         // Fine since are typed any.
         return;
     }
     
-    function f9(): void {
+    function f9(): any {
         // Fine since we are typed any and return undefined

```

Comment:
(To match the comments, as otherwise these two are exactly same with f5 and f6.)